src/main/java/com/softserve/academy/spaced/repetition/service/ImageService.java

```diff
@@ -51,10 +52,16 @@ public Image addImageToDB(MultipartFile file) throws ImageRepositorySizeQuotaExc
             throw new MultipartException("File upload error: file is too large.");
         } else {
             String base64 = encodeToBase64(file);
-            String imageType = file.getContentType();
-            image = new Image(base64, imageType, user, fileSize);
-            imageRepository.save(image);
-            image = imageRepository.getImageWithoutContent(image.getId());
+            if (!file.getContentType().split("/")[0].equals("image"))
```

Comment:
According to JavaDoc https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/multipart/MultipartFile.html#getContentType--
... null if not defined...
It means that a NullPointerException is possible here.